src/components/operations/operation-details/ko/runtime/operation-console.html

```diff
@@ -270,8 +270,19 @@ <h3>Headers</h3>
     <!-- ko if: hasBody -->
     <h3>Body</h3>
 
-    <div class="flex justify-content-end">
-        <!-- ko if: request.readonlyBodyFormat -->
+    <div class="row flex flex-row" style="margin-bottom: 3%;">
```

Comment:
Please avoid using `style` attribute in the code and rely on the styles. I think what you may want to checkout is `.form-group` class. Another things is 3% - which seems unreliable taking into account customers may use customization tools to adjust padding/margins.

Reply:
Thank you I have followed your instructions and you can see it now :)